sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule.models.ts

```diff
@@ -19,3 +22,8 @@ export enum EnrollmentPeriod {
   Current = "current",
   Upcoming = "upcoming",
 }
+
+export interface DisbursementScheduleWithStopFullTimeBCFundingStatus
+  extends DisbursementSchedule {
+  stopFullTimeBCFundingStatus: number;
```

Comment:
`Number` is not the proper way to map this.
Please consider changing to something like a boolean with the name `stopFullTimeBCFunding`